NAMESPACE

```diff
@@ -25,10 +25,12 @@ export(slackr_users)
 export(with_pagination)
 importFrom(cachem,cache_disk)
 importFrom(cachem,cache_mem)
+importFrom(dplyr,`%>%`)
```

Comment:
The pipe is already imported from `magrittr`, so you don't need to re-import it from `dplyr`.

With that said though, I'd actually prefer to clean this up a bit with `usethis::use_pipe()`. I'm happy to make that change, but if you wanted to drive-by fix it as part of this PR, that'd be great!

Reply:
I've used `usethis::use_pipe()` and removed pipe imports (`@importFrom ... %>%`).